ush/config.sh.RRFS_dev1

```diff
@@ -53,6 +54,17 @@ CYCLEMONTH="5-7"
 #CYCLEDAY="1-4"
 
 if [[ $DO_RETRO == "true" ]] ; then
+
+  if [[ $MACHINE == "jet" ]] ; then
+    EXTRN_MDL_SOURCE_BASEDIR_ICS=/mnt/lfs4/BMC/wrfruc/Ruifang.Li/data/hrrr/conus/wrfnat/grib2
```

Comment:
Do we have data staging scripts available? Or most users will get retro data from /mnt/lfs4/BMC/wrfruc/Ruifang.Li/data?

Reply:
Yes, it will make retro much easier if all the data for retros are under same directory. For May 2021 retro, all users should use data under  "/mnt/lfs4/BMC/wrfruc/Ruifang.Li/data" unless we decide to put the data in other location.

Reply:
Yes, we only needs one copy of the data in the disk and all retros should use this one copy of the data to save the disk space.